provisioning/device/typings.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "azure-device-provisioning-client",
+  "main": "index.d.ts",
+  "dependencies": {},
+  "globalDependencies": {
+    "node": "registry:dt/node#6.0.0+20160813124416"
```

Comment:
>node [](start = 5, length = 4)

no need for typings (history: typings are a deprecated way of getting type definitions and where possible have been replaced by @types/<package> packages. in that case we already have the @types/node package so no need for this.